playbook/app/pb_kits/playbook/pb_typeahead/_typeahead.scss

```diff
@@ -161,6 +161,11 @@
   .typeahead-plus-icon {
     color: $text_lt_lighter;
   }
+  .pb_badge_kit_primary {
+    margin-bottom: 2px;
+    margin-right: $space_xs;
+    margin-top: 2px;
+  }
```

Comment:
This is great @ElisaShapiro , any reason we aren't adding this margin as a global prop similar to what we do for the form pill? Also, was the top and bottom margin needed as well?